### DISCUSSION TITLE
Jede Phase einzeln auslesen am shelly 3em

### DISCUSSION THREAD

**Discussion by sponti68** [hoylabs/OpenDTU-OnBattery#1396] [Q&A]
Ich möchte an jeder Phase einen Wechselrichter mit jeweils eigener openDTU on battery betreiben.
Ziel / Grund: Auf jeder Phase nur das einspeisen was jeweils benötigt wird, wegen fehlender Phasensaldierung beim NB. etc.

Lese ich am shelly 3em den Status aus erhalte ich zwar alle drei Phasen einzeln, die über total_power in den Einstellungen der OpenDTU als Summe ausgegeben werden.

Muss ich dafür die json ändern?
Wie?

Oder gibt es auch die Möglichkeit in den Einstellungen der OpenDTU direkt statt "total_power" jede Phase einzeln auslesen zu lassen?

Wenn das irgendwie realisiert werden kann, dann möchte ich mit jeweils einem OpenDTU einen HM 1500 an jeder Phase betreiben.

Bin für jede Hilfe dankbar. 
Vielen Dank

**Comment by spcqike** [comment]
Guten Morgen,

> Muss ich dafür die json ändern?

ja

>  Wie?

> Oder gibt es auch die Möglichkeit in den Einstellungen der OpenDTU direkt statt "total_power" jede Phase einzeln auslesen zu lassen?

naja, das ist es halt.
Hast du dir den JSON einmal im Browser angeguckt und eventuell irgendwo weitere "power" Werte gefunden?

> Bin für jede Hilfe dankbar.

Tipp: die Kanalzahl muss im JSON als ``[x]`` geschrieben werden. also sowas wie ``abc/[x]/yxz"

Alternativ kannst du auch eine andere URL verwenden. Dann wird der JSON Pfad einfacher.
https://shelly-api-docs.shelly.cloud/gen1/#shelly-3em-emeter-index

**Reply by stefan123t** [reply]
I have described a [possible PM+DPL setup](https://github.com/tbnobody/OpenDTU/issues/272#issuecomment-2219644352) with a Shelly Pro3EM as **Power Meter (PM)** here in issue 272 upstream.

Please pay attention to the single phases:
* `em:0/a_act_power`
* `em:0/b_act_power`
* `em:0/c_act_power`
instead of the JSON Path for total_power:
* `em:0/total_act_power`
